sound/soc/sof/topology.c

```diff
@@ -1276,7 +1279,11 @@ static int sof_widget_load_dai(struct snd_soc_component *scomp, int index,
 	comp_dai.comp.hdr.size = sizeof(comp_dai);
 	comp_dai.comp.hdr.cmd = SOF_IPC_GLB_TPLG_MSG | SOF_IPC_TPLG_COMP_NEW;
 	comp_dai.comp.id = swidget->comp_id;
+#if IS_ENABLED(CONFIG_SND_SOC_SOF_VIRTIO_FE)
+	comp_dai.comp.type = SOF_COMP_VIRT_CON;
+#else
```

Comment:
Needs to be runtime configured based on operating mode (based on pdata).

Reply:
done